### ISSUE TITLE
Looking for new contributors

### ISSUE THREAD

**Issue by a-hacker** [ParameterizedBuilds/parameterized-builds#149]
I am the primary maintainer of this project. Lately, I have not much bandwidth to work on this plugin. In addition, my team moved off of Bitbucket so I haven't had as much motivation to work on new features.

I enjoy working on this project and will still act as maintainer but I need development help to cover the backlog of issues that have begun to pile up. If anyone is interested in being a contributor, please reach out to me over email (explosivepyrounicorns@gmail.com) or in our [Gitter chat](https://gitter.im/parameterized-builds/Lobby). I would greatly appreciate any help and will work to get any new contributors up to speed.

**Comment by kenbarby** [comment]
shoot me an mail. id be interested in contributing as we do still use this at our company

**Comment by a-hacker** [comment]
Hi @kenbarby,

Thanks for expressing interest in contributing! The first thing you should do is get your development environment set up. I wrote [some documentation in our wiki](https://github.com/KyleLNicholls/parameterized-builds/wiki/Developer-Guide) about how to do that.

Once you have your own set-up, I'd recommend looking through the open issues for something you'd like to do. I've marked several issues with "good first issues". Those should help you get familiar with the codebase. Once you have an issue you'd like to work on, send me an email and I'll assign it to you. 

If you have an questions about our code or getting set up, I'd be more than happy to provide assistance if you send me email. Thanks again!

**Comment by rockjing** [comment]
Our team is using your plugin of bitbucket, it's simple and work well for a long time. But recently after upgrade bitbucket to v6.3.0. It didn't work. So I tried to create a dev environment and see if the latest version could be packaged and run well on bitbucket v6.3.0. 

but I failed to download some jars which was customized by atlassian like：

>  Failed to execute goal on project parameterized-builds: Could not resolve dependencies for project com.kylenicholls.stash:parameterized-builds:atlassian-plugin:4.0.2: The following artifacts could not be resolved: org.codehaus.jackson:jackson-jaxrs:jar:1.9.13-atlassian-3, org.codehaus.jackson:jackson-xc:jar:1.9.13-atlassian-3: Could not find artifact org.codehaus.jackson:jackson-jaxrs:jar:1.9.13-atlassian-3 in nexus

org.codehaus.jackson:jackson-jaxrs:jar:**1.9.13-atlassian-3**

so how can I get these jar to setup dev?

**Comment by rockjing** [comment]
OK. I tried search on https://mvnrepository.com and I found the atlassian version - 1.9.13-atlassian-3 on that website. Let me try again. later to update.
